src/Search/Search.css

```diff
@@ -0,0 +1,17 @@
+.form {
+  display: flex;
+  justify-content: flex-end;
+  padding: 0.5em;
+  height: 5%;
+  font-family: inherit;
+}
+
+input {
+  padding: 1.5%;
+  font-size: 1.1em
+}
+
+.fa-search{
+  margin-top: 25%;
+  border-radius: 5px;
+}
```

Comment:
We can remove the border radius since this has been changed to an icon that doesn't have a border anymore.